### PR TITLE
Updated besman with 'bes modify' command

### PR DESCRIPTION
**Pull request by jobyko**
@asa1997 
Please share your feedback 

@sumodgeorge

### CODE REVIEW

File: src/main/bash/besman-mod.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+function __besman_validate_playbook_type
```
Comment (asa1997) [inline src/main/bash/besman-mod.sh:3]: This function is unnecessary

File: src/main/bash/besman-mod.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+function __bes_mod() {
+ local playbookdir="$HOME/$BESMAN_PLAYBOOK_REPO"
+ local filename=$1
+
+ if [[ ( -d "$playbookdir" ) && ( -f $playbookdir/$filename ) ]]; then
+         cd $playbookdir
+         __besman_validate_playbook_type $filename || return 1
+
+         __besman_vim_playbook $filename || return 1
```
Comment (asa1997) [inline src/main/bash/besman-mod.sh:21]: Please use jupyter notebook or vscode

**Review by asa1997 — CHANGES_REQUESTED** [review]
@jobyko Please address the comments

**Comment by jobyko** [comment]
Added the requested modification

**Pull request closed without merging**